File: output/StateMonadFFICheatcode.php
<?php
$exports['runStateMonadFFICheatcode'] = function($limit) {
    $state = 0;
    for ($i = 0; $i < 60; $i++) {
        for ($j = 0; $j < 20; $j++) {
            $state += 1;
        }
    }
    return $state;
};
return $exports;

File: run/bak/cpp/ffi/Test/StateMonadFFICheatcode.cc
#include "purescript.h"
namespace { volatile int dummy; }
FOREIGN_BEGIN( Test_StateMonadFFICheatcode )
exports["runStateMonadFFICheatcode"] = [](const boxed& in) -> boxed { 
    int n = unbox<int>(in); int state = 0;
    for(int i=0; i<n; i++) state++;
    dummy = state; return state;
};
FOREIGN_END
